lib/Rex/Interface/Connection/OpenSSH.pm

```diff
@@ -82,7 +82,10 @@ sub connect {
   $ssh_opts{ConnectTimeout} = $timeout;
 
   my %net_openssh_constructor_options = (
-    exists $ssh_opts{initialize_options} ? $ssh_opts{initialize_options} : () );
+    exists $ssh_opts{initialize_options} ? %{ $ssh_opts{initialize_options} } : () );
```

Comment:
I'm not sure how this ended up as part of this PR - can you rebase your branch on top of current master, and push again, please?